docs/reference/connector/apis/connector-apis.asciidoc

```diff
@@ -3,17 +3,16 @@
 
 preview::[]
 
-++++
-<titleabbrev>Connector APIs</titleabbrev>
-++++
-
----
-
-The connector and sync jobs API provides a convenient way to create and manage Elastic connectors and sync jobs in an internal index.
+The connector and sync jobs API provides a convenient way to create and manage Elastic {enterprise-search-ref}/connectors.html[connectors^] and sync jobs in an internal index.
 
 This API provides an alternative to relying solely on {kib} UI for connector and sync job management. The API comes with a set of
 validations and assertions to ensure that the state representation in the internal index remains valid.
 
+[TIP]
+====
+Use our https://github.com/elastic/connectors/blob/main/docs/CLI.md[CLI] to manage connectors and sync jobs programmatically.
```

Comment:
I feel like this tip is provided without necessary context. 

We start this doc page with 

> The connector and sync jobs API provides a convenient way to create and manage Elastic connectors and sync jobs

but then the tip says to use CLI. I'm confused what we are trying to communicate here.

Reply:
Rather than "programatic" I would say sth:
> We also offer connector client to manage connectors and sync jobs through a command-line interface

Reply:
@jedrazb the idea is really to just make folks aware of the CLI option, but if you feel like it's not super helpful we can remove it for the moment. Let me know what you think.

Reply:
Updated to this in meantime, so it's more of just a signpost without confusing verbiage:

```We also have a command-line interface for Elastic connectors. Learn more in the https://github.com/elastic/connectors/blob/main/docs/CLI.md[elastic/connectors] repository.```

Reply:
That's much better IMO, since we are explicitly saying it's command line interface